easybuild/easyconfigs/p/PSI/PSI-4.0b6-3815c4672b2-intel-2015a-mt.eb

```diff
@@ -0,0 +1,29 @@
+name = 'PSI'
+commit = '3815c4672b2'
```

Comment:
use a datestamp as a versionsuffix rather a meaningless/ugly commit ID in the module name?